packages/css/src/index.css

```
@@ -1,13 +1,13 @@
 @charset 'UTF-8';
 
-@layer ds.base, ds.typography, ds.theme, ds.components;
+@layer ds.base, ds.theme, ds.components;
 
 /** Import order defines ordinal specificity for layers */
 @import url('./base.css') layer(ds.base);
-@import url('./heading.css') layer(ds.typography);
-@import url('./label.css') layer(ds.typography);
-@import url('./paragraph.css') layer(ds.typography);
-@import url('./validation-message.css') layer(ds.typography);
+@import url('./heading.css') layer(ds.components);
+@import url('./label.css') layer(ds.components);
+@import url('./paragraph.css') layer(ds.components);
+@import url('./validation-message.css') layer(ds.components);
```

```suggestion
@import url('./validation-message.css') layer(ds.components);
/** Typography and base should always come first */
```

Can we add a comment that it should not be moved?

---

I just moved them them to the `ds.base` layer instead, as we also had problem with selectors inside `Card` and `Paragraph` when overriding margins. 

Them being put in the `ds.base` layer should be indication enough that they always come first.